static/js/tools/timeline_util.ts

```diff
@@ -274,6 +274,72 @@ function setSearchParam(vars: VarUrl) {
   window.location.hash = newHash;
 }
 
+interface StatsVarNode {
```

Comment:
please add some comments for the interface and class

Reply:
done